examples/README.md

```diff
@@ -3,7 +3,19 @@
 Examples demonstrating how to interact with Eigen layer contracts using eigen-rs crates.
 
 ## OperatorsInfo
-
 ```
 cargo run -p info-operator-service
 ```
+
+## AvsRegistry Read
+```
+cargo run -p examples-avsregistry-read
```

Comment:
can you add some test as to what these do?

Reply:
Made a push , the command was wrong . wdym by test?

Reply:
same, text